src/styles/styles.js

```diff
@@ -0,0 +1,34 @@
+export const styles = {
+  colors: {
+    gray1: "rgb(45, 45, 45)",
```

Comment:
more specified colors would be better so that it make sense such as lightGray, darkGray, limeGray and so on, you can't know the colors scale with just a number. I prefer the variable name to be longer and descriptive thant generic and non readable, **this applies to all variable names** in the code